interface/migrations/0009_auto_20191101_2102.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.6 on 2019-11-01 19:02
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('interface', '0008_submission_penalty'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='assignment',
+            old_name='deadline',
+            new_name='deadline_soft',
+        ),
+        migrations.AddField(
+            model_name='assignment',
+            name='deadline_hard',
+            field=models.DateTimeField(null=True),
+        ),
+        migrations.AlterField(
+            model_name='submission',
+            name='assignment',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='interface.Assignment'),
```

Comment:
I've fixed this in https://github.com/vmck/acs-interface/commit/d266ea645c948de5ebbd26c0c8af0a068b8056ae